fontforgeexe/collabclientui.c

```diff
@@ -433,8 +433,8 @@ static void collab_sessioncallbacks_init (CollabSessionCallbacks *self) {}
 static void
 collab_sessioncallbacks_class_init (CollabSessionCallbacksClass *klass)
 {
-    GType argtypes[] = { G_TYPE_POINTER, NULL };
-    
+    GType argtypes[] = {G_TYPE_POINTER};
```

Comment:
`GType` is not a pointer type, so using `NULL` in this case doesn't make sense. Usually lists in GLib need to be NULL terminated, but in this case it doesn't need to be because the size parameter is specified in `g_signal_newv`.